app/src/main/java/org/wikipedia/auth/AccountUtil.kt

```diff
@@ -102,6 +106,17 @@ object AccountUtil {
         return SharedPreferenceCookieManager.instance.getCookieExpiryByName(CENTRALAUTH_USER_COOKIE_NAME)
     }
 
+    fun maybeShowTempAccountWelcome(activity: Activity) {
```

Comment:
Does the temp account work logic work for the article description? If the user posts an article description in `enwiki`, does the app show maybeShowTempAccountWelcome for the user?

Same question for Commons and Wikidata.

Reply:
done!